src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1523,8 +1523,9 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
     }
 
     void visit_ListItem(const ASR::ListItem_t& x) {
-        ASR::ttype_t* el_type = ASRUtils::get_contained_type(
```

Comment:
Does this fails?

I think `get_contained_type` already handles `Const`

Reply:
We need this @Thirumalai-Shaktivel as without it the element type (el_type) will be set to `list` for `Const`. We need it to store the value inside the `Const list`.